examples/cloudml-churn-prediction/preprocessor/run_preprocessing.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Start preprocessing job for Survival Analysis TFRecords."""
+
+import logging
```

Comment:
IMO this doesn't warrant it's own file move this into `preprocess.py`

Reply:
This is following the juliaset beam example, which is generally used for a template for Python on Dataflow. https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/complete/juliaset/juliaset_main.py

I'm fine with moving it into `preprocess.py`, just want to make sure it's following best practices. Should I still move it?

Reply:
Touche, Not sure why that example is like that.
You'll see other example pipelines don't do that ie. the canonical wordcount example https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/wordcount.py#L114
As far as best practice here, I think it's a bit ambiguous I just think it's extra files for no real added value which is why I said IMO.
I'll leave it up to you.